R/annotate_cn_matrix_by_genes.R

```diff
@@ -0,0 +1,53 @@
+#' @title Annotate copy number matrix by gene names
+#' 
+#' @description Annotates a copy number matrix using user-provided gene names 
+#' 
+#' @param gene_ids The "gene_id" column stores gene symbols (characters).
```

Comment:
The documentation should be more explicit about what happens in the function. If the input has regions that don't contain any of the user-specified genes, are those columns dropped? How does it determine the locations of each gene? What if a gene spans more than one region etc.